en/micro-integrator/docs/setup/deployment/k8s-pipeline/modifying-pipeline-configurations.md

```diff
@@ -0,0 +1,201 @@
+# Modifying Pipeline Configurations
+
+## The pipeline configurations can be changed by the modification of values.yaml file
+
+The pipeline configurations can be changed by the modification of the
+configurations in
+the [values.yaml](https://github.com/wso2/kubernetes-pipeline/blob/master/kubernetes-pipeline/values.yaml) file
+corresponding to the pipeline. The [sample
+modifications](https://github.com/wso2/kubernetes-pipeline/tree/master/kubernetes-pipeline/samples)
+for each product provide a good indication as to how these
+configurations could be modified upon deployment. Once a configuration
+change has been made, run the helm upgrade command to apply the changes.
+
+``` bash
+$ helm upgrade <RELEASE_NAME> wso2/kubernetes-pipeline --version 1.0.0 -f values.yaml
+```
+>
+
+**<RELEASE\_NAME\>** should be replaced with the release name provided
+when the pipeline is installed initially.
+
+Configuration changes occur which will affect the Jenkins
```

Comment:
```suggestion
Configuration changes occur that will affect the Jenkins
```

You'd normally have a comma before "which" and none before "that". The reasoning is when you use "which" you are adding information that is additional to the main sentence. When you use "that", it means the information after that is necessary for the sentence.